addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_edi.tests.common import AccountEdiTestCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestAccountEdiUblCii(AccountEdiTestCommon):
```

Comment:
Could you rather inherit `AccountTestInvoicingCommon`, please ? :) `account_edi_ubl_cii` does no longer depend on `account_edi` in saas-16.2